.github/workflows/ci.yml

```diff
@@ -31,10 +31,10 @@ jobs:
         -   uses: actions/checkout@v2
         -   name: "Main Script"
             run: |
-                set -e
                 sed 's/python=3/python=2.7/' .test-conda-env-py3.yml > .test-conda-env-py2.yml
                 CONDA_ENVIRONMENT=.test-conda-env-py2.yml
                 curl -L -O -k https://gitlab.tiker.net/inducer/ci-support/raw/master/build-and-test-py-project-within-miniconda.sh
+                python ./configure.py --cl-inc-dir="$CONDA_PREFIX/include" --cl-lib-dir="$CONDA_PREFIX/lib" --yoink
```

Comment:
CONDA_PREFIX env variable is not set here

Reply:
Ouch. Thanks!

Reply:
797337a, maybe. :)